src/fields.ts

```diff
@@ -26,40 +29,79 @@ export const enum RichTextNodeType {
 // Text nodes
 
 /**
- * @internal
+ * Base to be extended by other RT Nodes.
  */
 export interface RTTextNodeBase {
 	text: string;
 	spans: RTInlineNode[];
 }
 
+/**
+ * Rich Text <h1 /> node
```

Comment:
`<h1 />` is an HTML-specific identifier. Rich Text could be serialized to non-HTML elements, like a React component or something complete differently, like a SwiftUI View instance on iOS.

Something like this is sufficient:

```
Rich Text `heading1` node.
```